<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>structure</class>
 <widget class="QWidget" name="structure">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit structure</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>28</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lamda</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>97</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit structure</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Structure 1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Structure 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="colorButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>107</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string extracomment="Select type of potential for interactions between items in structure 1">Potential for structure 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coulomb potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debye-Hückel potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dipole-dipole potential</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>133</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string extracomment="Select type of potential for interactions between items in structure 1">Potential for structure 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coulomb potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debye-Hückel potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dipole-dipole potential</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>159</y>
     <width>137</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string extracomment="Select type of potential for interactions between items in structure 1">Potential for 1-2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coulomb potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debye-Hückel potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dipole-dipole potential</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>231</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomOutButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomInButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>289</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>318</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>356</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>385</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>structure</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
